noir/noir-repo/noir_stdlib/src/grumpkin_scalar_mul.nr

```diff
@@ -1,7 +1,10 @@
 use crate::grumpkin_scalar::GrumpkinScalar;
-use crate::scalar_mul::fixed_base_embedded_curve;
+use crate::scalar_mul::{fixed_base_embedded_curve, variable_base_embedded_curve};
 
 pub fn grumpkin_fixed_base(scalar: GrumpkinScalar) -> [Field; 2] {
-    // TODO: this should use both the low and high limbs to do the scalar multiplication
     fixed_base_embedded_curve(scalar.low, scalar.high)
 }
+
+pub fn grumpkin_variable_base(point_x: Field, point_y: Field, scalar: GrumpkinScalar) -> [Field; 2] {
+    variable_base_embedded_curve(point_x, point_y, scalar.low, scalar.high)
+}
```

Comment:
```suggestion
```

We'd like to remove this file in future so we should avoid adding more entries to it.